documentation/src/main/markdown/releasenotes.md

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
What is the conflict in this file (according to github) ?

Reply:
My git diff showed the files were identical so I'm not sure why ... in any case, the "conflicts" are now resolved 👍